src/main/java/io/workflow/bpmnsimulator/fieldvalidator/AssigneeFieldValidator.java

```diff
@@ -0,0 +1,29 @@
+package io.workflow.bpmnsimulator.fieldvalidator;
+
+import io.workflow.bpmnsimulator.model.Field;
+import io.workflow.bpmnsimulator.model.Step;
+import lombok.extern.slf4j.Slf4j;
+import org.camunda.bpm.engine.task.Task;
+import org.springframework.stereotype.Service;
+
+import javax.annotation.Nonnull;
+import java.util.Objects;
+
+@Slf4j
+@Service
+class AssigneeFieldValidator implements FieldValidator {
```

Comment:
can be rename to AssigneeValidator

Reply:
Sure. It's done here: https://github.com/mohsen-ebrahimi/bpmn-simulator/pull/3